app/components/taskLogs/taskLogsController.js

```diff
@@ -0,0 +1,75 @@
+angular.module('taskLogs', [])
+.controller('TaskLogsController', ['$scope', '$stateParams', '$anchorScroll', '$q', 'ServiceService', 'TaskService', 'TaskLogs', 'Task', 'Notifications',
+function ($scope, $stateParams, $anchorScroll, $q, ServiceService, TaskService, TaskLogs, Task, Notifications) {
```

Comment:
`$anchorScroll` and `$q` is not being used, we can remove them.

Reply:
Removed